File: Java Food Order App/Customer.java
package FoodOrderApp;

public class Customer {
	byte customerID;
	Name customerName;
	Address customerAddress;
	long customerPhone;
	String customerEmail;
	
	Customer (byte id, Name name, long phone, Address address, String email) {
		this.customerID = id;
		this.customerName = name;
		this.customerAddress = address;
		this.customerPhone = phone;
		this.customerEmail = email;
	}
	
	void display() {
		System.out.println(customerID + "\t" + customerName + "\t" + customerPhone + "\t" + customerAddress + "\t" + customerEmail);
	}
	
}

File: Kotlin Food Order App/Customer.kt
package foodOrderApp

open class Customer(customerID: Byte, customerName: Name, customerPhone: Long, customerAddress: Address, customerEmail: String) {
    var customerID: Byte = 0
    var customerName: Name
    var customerPhone: Long = 0L
    var customerAddress: Address
    var customerEmail: String? = null
    init{
        this.customerID = customerID
        this.customerName = customerName
        this.customerPhone = customerPhone
        this.customerAddress = customerAddress
        this.customerEmail = customerEmail
    }
    fun display() {
        println("$customerID\t$customerName\t$customerPhone\t$customerAddress\t$customerEmail")
    }
}